internal/provider/resource_scalr_environment.go

```diff
@@ -73,15 +72,6 @@ func resourceScalrEnvironment() *schema.Resource {
 				DefaultFunc: scalrAccountIDDefaultFunc,
 				ForceNew:    true,
 			},
-			"policy_groups": {
```

Comment:
@denkl  If we still have policy_groups in the environment datasource, why won't we keep it as readonly in the resource as well?

Reply:
@petroprotsakh that's correct, fixed